sirius_control/sirius_control/sirius_status.py

```diff
@@ -0,0 +1,83 @@
+import rclpy
+from rclpy.node import Node
+from industrial_msgs.msg import RobotStatus, TriState, RobotMode
+from std_msgs.msg import String
+
+
+class SiriusStatus(Node):
+
+    def __init__(self):
+        super().__init__('sirius_status')
+        self.send_status = '/sirius_status'
+        self.receive_topics = '/topic_received'
+        self.initialize_robot_status()
+        self.create_subscriptions()
+        self.publisher = self.create_publisher(RobotStatus, self.send_status, 10)
```

Comment:
```suggestion
        self.initialize_robot_status()
        self.publisher = self.create_publisher(RobotStatus, 'robot_status', 10)
        self.joy_subscription_ = self.create_subscription(
        String, self.receive_topics, 'joy_multiplexer/selected_output', 10)
        self.middleware_subscription_ = self.create_subscription(
        String, 'relaxing_middleware/state', self.relaxing_middleware_callback, 10)
```
There are multiple changes:
- Removed `send_status` and `receive_topics` variables. They are used only once when creating publisher and subscribers. IMO it's cleaner if we put those names directly in the publisher/subscriber declarations.
- Added subscribers declarations in here and removed call to create_subscribtion method. IMO it's better to see all publisher/subscriber declarations in one place, especially that there are not a lot of them. Though it's just my personal opinion, so If you prefer you can leave it as it is - encapsulated in `create_subscriptions` method. 
- Changed subscriber topics to be distinctive, so they can actually listen to different topics. 
- Removed leading `/` from topic names. It would force topics to be global - they would not be pushed into a namespaced together with the node. You can read about the concept of relative/global/private names in [ROS 1 wiki](http://wiki.ros.org/Names). As a rule of thumb in future use private names for parameters and relative names for everything else.